"make test-integration" fails

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
BUG REPORT

**Kubernetes version** (use `kubectl version`):
version.Info{Major:"1", Minor:"6+", GitVersion:"v1.6.0-alpha.0.871+1dfd64f4378ad9-dirty", GitCommit:"1dfd64f4378ad9dd974bbfbef8e90127dce6aafe", GitTreeState:"dirty", BuildDate:"2016-11-24T09:20:15Z", GoVersion:"go1.7.3", Compiler:"gc", Platform:"linux/amd64"}

**Environment**:
- **Cloud provider or hardware configuration**:
Local machine.
- **OS** (e.g. from /etc/os-release):
Ubuntu 16.04
- **Kernel** (e.g. `uname -a`):
Linux fujitsu 4.4.0-45-generic #66-Ubuntu SMP Wed Oct 19 14:12:37 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux


**What happened**:
"make test-integration" fails with such output:
#make test-integration
+++ [1124 10:03:58] Generating bindata:
    /home/fujitsu/go/src/k8s.io/kubernetes/test/e2e/framework/gobindata_util.go
+++ [1124 10:03:59] Building the toolchain targets:
    k8s.io/kubernetes/hack/cmd/teststale
+++ [1124 10:03:59] Building go targets for linux/amd64:
    cmd/libs/go2idl/conversion-gen
+++ [1124 10:04:16] Generating bindata:
    /home/fujitsu/go/src/k8s.io/kubernetes/test/e2e/framework/gobindata_util.go
+++ [1124 10:04:17] Building the toolchain targets:
    k8s.io/kubernetes/hack/cmd/teststale
+++ [1124 10:04:17] Building go targets for linux/amd64:
    cmd/libs/go2idl/openapi-gen
+++ [1124 10:04:34] Checking etcd is on PATH
/usr/bin/etcd
+++ [1124 10:04:34] Starting etcd instance
etcd --advertise-client-urls http://127.0.0.1:2379 --data-dir /tmp/tmp.Xj6deJvgyg --listen-client-urls http://127.0.0.1:2379 --debug > "/dev/null" 2>/dev/null
Waiting for etcd to come up.
+++ [1124 10:04:34] On try 2, etcd: : http://127.0.0.1:2379
{"action":"set","node":{"key":"/_test","value":"","modifiedIndex":4,"createdIndex":4}}
+++ [1124 10:04:34] Running integration test cases
+++ [1124 10:04:44] Generating bindata:
    /home/fujitsu/go/src/k8s.io/kubernetes/test/e2e/framework/gobindata_util.go
+++ [1124 10:04:44] Building the toolchain targets:
    k8s.io/kubernetes/hack/cmd/teststale
flag provided but not defined: -short
usage: install [build flags] [packages]

Install compiles and installs the packages named by the import paths,
along with their dependencies.

For more about the build flags, see 'go help build'.
For more about specifying packages, see 'go help packages'.

See also: go build, go get, go clean.
Makefile.generated_files:757: recipe for target '_output/bin/conversion-gen' failed
make[2]: *** [_output/bin/conversion-gen] Error 1
Makefile:296: recipe for target 'generated_files' failed
make[1]: *** [generated_files] Error 2
+++ [1124 10:04:44] Cleaning up etcd
+++ [1124 10:04:44] Integration test cleanup complete
Makefile:138: recipe for target 'test-integration' failed
make: *** [test-integration] Error 1

**What you expected to happen**:
No error happened.

**How to reproduce it** (as minimally and precisely as possible):
#make test-integration

--- a/hack/make-rules/test-integration.sh
+++ b/hack/make-rules/test-integration.sh
@@ -62,8 +62,8 @@ runTests() {
   # KUBE_RACE="-race"
   make -C "${KUBE_ROOT}" test \
       WHAT="$(kube::test::find_integration_test_dirs ${2-} | paste -sd' ' -) $(echo ${@:3})" \
-      KUBE_GOFLAGS="${KUBE_GOFLAGS:-} ${SHORT:--short=true} -tags 'integration no-docker'" \
-      KUBE_TEST_ARGS="${KUBE_TEST_ARGS:-} --vmodule=garbage*collector*=6 --alsologtostderr=true" \
+      KUBE_GOFLAGS="${KUBE_GOFLAGS:-} -tags 'integration no-docker'" \
+      KUBE_TEST_ARGS="${KUBE_TEST_ARGS:-} ${SHORT:--short=true} --vmodule=garbage*collector*=6 --alsologtostderr=true" \
       KUBE_RACE="" \
       KUBE_TIMEOUT="${KUBE_TIMEOUT}" \
       KUBE_TEST_API_VERSIONS="$1"

